core/src/main/java/com/adyen/checkout/core/components/CheckoutConfiguration.kt

```diff
@@ -155,3 +161,19 @@ class CheckoutConfiguration(
         }
     }
 }
+
+private fun Parcel.writeNullableBoolean(value: Boolean?) {
```

Comment:
I think we can remove `nullable` part from the function name for these two functions to align with other functions such as `parcel.readString()` and `parcel.readSerializable()` which are nullable functions.